docs/images/esm.svg

```diff
@@ -14,7 +14,7 @@
   <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
     <text x="16.5" y="15" fill="#010101" fill-opacity=".3">esm</text>
     <text x="16.5" y="14">esm</text>
-    <text x="59" y="15" fill="#010101" fill-opacity=".3">4.31 kB</text>
-    <text x="59" y="14">4.31 kB</text>
+    <text x="59" y="15" fill="#010101" fill-opacity=".3">9.48 kB</text>
+    <text x="59" y="14">9.48 kB</text>
```

Comment:
this doubled because of the JSDocs, but shouldn't matter, because this is the server (check the UMD file for web impact)